src/lib/flexbox/api/base.ts

```diff
@@ -127,7 +135,7 @@ export abstract class BaseFxDirective implements OnDestroy, OnChanges {
     let value = this._lookupStyle(element, 'display');
 
     // Note: 'inline' is the default of all elements, unless UA stylesheet overrides
-    return value ? value.trim() : 'inline';
+    return value ? value.trim() : ((element.nodeType === 1) ? 'block' : 'inline');
```

Comment:
`_lookupStyle()` checks in-line styles or call `getComputedStyle()`. So the expected response usually be defined.  

I think `return value ? value.trim() : 'block'` is a reasonable fallback when `_lookupStyle()` fails.